CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+
+# Mirroring the images
+
+If the external release image registry is not accessing in your setup all images can be mirrored 
+to a different registry using `mirror` subcommand:
+```
+oc adm -a /path/to/pull-secret.json \
+  release mirror \
+  --from "registry.svc.ci.openshift.org/origin/release:4.4.0-0.okd-2020-03-13-053843" \
+  --to quay.io/vrutkovs/okd-content \
+  --to-release-image quay.io/vrutkovs/okd-release:4.4
+```
+This command would copy images, referenced in `registry.svc.ci.openshift.org/origin/release:4.4.0-0.okd-2020-03-13-053843` image to `quay.io/vrutkovs/okd-content` and create a new release image `quay.io/vrutkovs/okd-release:4.4`. The new release image would use `quay.io/vrutkovs/okd-content` as an 
+additional source of images.
+
+`release mirror` command would also print out the mirroring configuration for the installer.
+
+See https://docs.okd.io/latest/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html
+
+# Making changes
+
+`oc adm release` can be used to create new releases, amending existing releases. If one of the images 
+needs to be updated use `release new` subcommand.
+
+For example, lets create a new release with updated Prometheus image. Use `release info` command 
+to print the pullspec used in the existing image:
+```
+$ oc adm release info registry.svc.ci.openshift.org/origin/release:4.4 --pullspecs | grep prometheus
+  k8s-prometheus-adapter                         registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:12bac47c71cb7ef36b6ee7b78e0476fbfb8a67bbf61ac42c461c17c98ac850a6
+  prometheus                                     registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:5af0373659974782379d90d9a174352dd8f85cb7327cc48ef36cae4e8ba5903f
+  prometheus-alertmanager                        registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:25bed531ccb0ff16ce19b927265f03cb9b2d572caa224ef302002269e925d83c
+  prometheus-config-reloader                     registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:deacbd618b3c037cc8c99a83db2c2a1053db517b0a0bfdfdeb309591559c3eea
+  prometheus-node-exporter                       registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:c199e7353642ed1a4237416055a75b0e415034c7ec48bbc8ae8d12b72552f819
+  prometheus-operator                            registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:ec28b9dc5ad9184d0d70b85e5bc618c809084b293cbc57c215bf845bf7147b2b
+```
+
+In `release new` command a list of replaced images can be specified:
+```
+$ oc adm -a /path/to/pull_secret.json \
+  release new \
+  --from-release registry.svc.ci.openshift.org/origin/release:4.4 \
+  --to-image quay.io/vrutkovs/okd-release:4.4-updated-prometheus \
+  prometheus=docker.io/prom/prometheus:v2.17.1
```

Comment:
q - could you please explain (it might be obvious however i miss s'thing) where is the relation between `--from-release` and the `release info` output above? In other words, after we have the pullspec for prometheus ( very good example btw which shows more than one pullspec related to it) which pullspec needs the user to pull and how that is reflected into `release new` command?

Reply:
Right, good idea, let me expand that